python/code/classificationmethods/ManifoldDiscrimantAnalysisParafacTucker.py

```diff
@@ -0,0 +1,193 @@
+from abc import ABC, abstractmethod
+import numpy as np
+from scipy.stats import ortho_group  
+from pymanopt import Problem
+from pymanopt.manifolds import Product, Stiefel
+import tensorflow as tf
+from pymanopt.solvers import ConjugateGradient
+from sklearn.pipeline import Pipeline
+
+
+class ManifoldDiscrimantAnalysis(ABC, Pipeline):
+    def __init__(self, classes=None, opts=None, usestoppingcrit=True, maxits=1000,
+                 store={'Rw': None, 'Rb': None, 'QtRb': None, 'QtRw':  None, 'QtBQ': None, 'QtWQ': None, 'QtWQinvQtBQ': None},
+                 optmeth='ManOpt'):
+        self.classes = classes
```

Comment:
Do we need classes as a Class variable? Shouldn't we pass it directly to the fit method?